### PR TITLE
Update Microsoft.NET.Test.Sdk package to latest version 16.10.0

### PR DESCRIPTION
**Pull request by utpilla**
## Changes

- Update Microsoft.NET.Test.Sdk package to latest version 16.10.0

### CODE REVIEW

**Comment by eddynaka** [comment]
@utpilla , u need to change here as well: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/build/process-codecoverage.ps1

**Comment by eddynaka** [comment]
> @utpilla , u need to change here as well: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/build/process-codecoverage.ps1

I would also change this package: MicrosoftCodeCoveragePkgVer

those normally are released together.

**Review by eddynaka — APPROVED** [review]

File: build/Common.nonprod.props [hunk 1]
```diff
@@ -36,7 +36,7 @@
     <MicrosoftExtensionsLoggingPkgVer>[5.0.0,6.0)</MicrosoftExtensionsLoggingPkgVer>
     <MicrosoftExtensionsLoggingAbstractionsPkgVer>[5.0.0,6.0)</MicrosoftExtensionsLoggingAbstractionsPkgVer>
     <MicrosoftExtensionsLoggingConsolePkgVer>[5.0.0,6.0)</MicrosoftExtensionsLoggingConsolePkgVer>
-    <MicrosoftNETTestSdkPkgVer>[16.9.1]</MicrosoftNETTestSdkPkgVer>
+    <MicrosoftNETTestSdkPkgVer>[16.10.0]</MicrosoftNETTestSdkPkgVer>
```
Comment (cijothomas) [inline build/Common.nonprod.props:39]: can you put a comment above this line to remind about updating the codecoverage.ps1 file as well when updating this?

Comment (utpilla) [reply build/Common.nonprod.props:39]: I have added a comment in the `Common.props` file where we have the PackageReference for `Microsoft.CodeCoverage`

Comment (reyang) [reply build/Common.nonprod.props:39]: Is it possible to update PowerShell code to "fetch" the version from the `Common.props` instead of hardcode?

**Review by cijothomas — APPROVED** [review]

**Pull request merged**